packages/fxa-auth-client/package.json

```diff
@@ -11,7 +11,9 @@
   "scripts": {
     "postinstall": "(tsc --build tsconfig.browser.json && tsc --build) || true",
     "build": "tsc --build tsconfig.browser.json && tsc --build",
-    "test": "mocha -r esbuild-register test/*"
+    "test": "mocha -r esbuild-register test/*",
+    "test:unit": "yarn test",
```

Comment:
Should we invert this to `"test:unit": "mocha -r esbuild-register test/*"`, and then modify `test` to something like `"test": "CI=true npm-run-all test:*"`?